tests/integration/l1l3Bridger.test.ts

```diff
@@ -356,7 +366,16 @@ describe('L1 to L3 Bridging', () => {
 
     itOnlyWhenCustomGasToken(
       'should properly get l2 and l1 fee token addresses',
-      async () => {
+      async function () {
+        const decimals = await getNativeTokenDecimals({
+          l1Provider: l1Signer.provider!,
+          l2Network: l3Network,
+        })
+
+        if (decimals !== 18) {
+          this.skip()
```

Comment:
For non-18 decimals these tests throw: `Error: L1 gas token has incorrect decimals. Use skipGasToken when depositing`.

We skip all the tests that throw this error for non-18 decimals.